unittests/Syntax/StmtSyntaxTests.cpp

```diff
@@ -243,32 +256,36 @@ TEST(StmtSyntaxTests, ReturnStmtGetAPIs) {
 }
 
 TEST(StmtSyntaxTests, ReturnStmtWithAPIs) {
-  auto ReturnKW = SyntaxFactory::makeReturnKeyword("", " ");
-  auto Minus = SyntaxFactory::makePrefixOperator("-", "", "");
-  auto OneDigits = SyntaxFactory::makeIntegerLiteral("1", "", "");
-  auto MinusOne = SyntaxFactory::makePrefixOperatorExpr(Minus,
-    SyntaxFactory::makeIntegerLiteralExpr(OneDigits));
+  RC<SyntaxArena> Arena = SyntaxArena::make();
+  auto ReturnKW = SyntaxFactory::makeReturnKeyword("", " ", Arena);
```

Comment:
Not for this PR, but maybe it's time we make `SyntaxFactory`'s methods instance methods and have it own the area in which new nodes are created...

Reply:
Sounds like it might be a good idea. I’ll take a look at it later.